lib/http2/hpack.c

```diff
@@ -264,6 +258,11 @@ static inline struct st_h2o_hpack_header_table_entry_t *header_table_get(h2o_hpa
     return entry;
 }
 
+inline h2o_hpack_header_table_entry_t *h2o_hpack_header_table_get(h2o_hpack_header_table_t *header_table, size_t index)
+{
+    return (h2o_hpack_header_table_entry_t*)(header_table_get(header_table, index));
+}
+
```

Comment:
All `inline` functions must be also defined as `static`.

Also, I would suggest refraining from creating bridges (i.e. a function that just calls another function), since it makes the code-base unnecessary complex. Our general preference goes to keeping the code-base compact and precise, rather than minimizing impact of each PR.

In this case, I would suggest changing `static inline struct st_h2o_hpack_header_table_entry_t *header_table_get(...)` to `h2o_hpack_eader_table_entry_t *h2o_hpack_header_table_get(...)`. Even without `inline` attribute, compilers are allowed to (and will) inline-expand functions defined in the same source file as the caller.